### DISCUSSION TITLE
How to display data sequenced on a timeline

### DISCUSSION THREAD

**Discussion by lildots** [gephi/gephi#2785] [Q&A]
Hi folks,
I work for a university scheduling exams. I managed to create a network diagram of each of our exams to be scheduled, and their co-enrolments as edges... it produced a great diagram conveying the complexity of the enrolment structure and why the scheduling is such a job.

I'd like to create a diagram that shows the exams on a timeline once they've been scheduled, something a little bit like the screenshot where each vertical series represents the day of the exam and what subjects are scheduled for that day, with each subsequent day and its exams shown on a timeline. I haven't found a plugin that will let me simply sort the data by a parameter across a horizontal or vertical row. I'm hoping to do this to demonstrate why it's not so easy to "move my subject to Monday!" because of the clashes it will cause.

Is there an easy way to do this?

![Capture](https://user-images.githubusercontent.com/129234678/228390853-6c0bb6a9-0667-467a-a001-83ed6de5fb31.PNG)

Thank you for any advice :)

**Comment by mbastian** [comment]
Hi @lildots I think the [Position Ranking Plugin](https://gephi.org/plugins/#/plugin/position-ranking-plugin) is what you're looking for.
